### DISCUSSION TITLE
Autoclosing of modified PR

### DISCUSSION THREAD

**Discussion by napei** [renovatebot/renovate#8633] [Request Help] [answered]
Is it at all configurable to ignore changes to the branch and close anyway?

![image](https://user-images.githubusercontent.com/8946502/107588499-f0a17600-6c57-11eb-88fd-19fec0812e1d.png)

In my case, it's just a script that updates additional checksum files and runs gazelle (bazel) to update build files. Would be great to be able to explicitly say "you can autoclose these if they're modified".

**Comment by viceice** [comment, marked as answer]
If your script always runs after renovate updates a pr, you can reuse the renovate git author, so renovate will think it's his own commit and can overwrite it.

**Reply by napei** [reply]
Thanks for that. It would be good if renovate could be configured to ignore specific authors.

**Reply by rarkins** [reply]
If you're interested in implementing it, please raise a feature request issue first so we can find an elegant approach.
